src/fusets/openeo/services/descriptions/phenology.md

```diff
@@ -60,7 +60,7 @@ s2 = connection.load_collection('SENTINEL2_L2A_SENTINELHUB',
                                 spatial_extent=spat_ext,
                                 temporal_extent=temp_ext,
                                 bands=["B04", "B08", "SCL"])
-s2 = s2.process("mask_scl_dilation", data=s2, scl_band_name="SCL")
+s2 = s2.process("mask_scl_dilation", data=s2, scl_band_name="SCL") # do md file also need updating?
```

Comment:
There are md file with the mask_scl_dilation process, do they also need to be updated

Reply:
I would say yes. These MD files serve as documentation pages for websites. Maybe best to also check with Jeroen